campari/io.py

```diff
@@ -0,0 +1,444 @@
+# Standard Library
+# Common Library
+import glob
+import os
+import pathlib
+import warnings
+
+import healpy as hp
+import numpy as np
+import pandas as pd
+import yaml
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.table import QTable, hstack
+from astropy.utils.exceptions import AstropyWarning
+from erfa import ErfaWarning
```

Comment:
I would put `numpy`, `pandas`, and `yaml` in their own block.  Widely used outside of astronomy.

Then the `astropy` and `erfa` are the astro-specific.

And then I would write it as
```
import astropy.units as u
```
and alphabetize after the `from astropy.utils...` line